Index: .idea/.idea.Swashbuckler.dir/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\n/shelf/\n/workspace.xml\n# Rider ignored files\n/projectSettingsUpdater.xml\n/contentModel.xml\n/.idea.Swashbuckler.iml\n/modules.xml\n# Editor-based HTTP Client requests\n/httpRequests/\n# Datasource local storage ignored files\n/dataSources/\n/dataSources.local.xml\n
===================================================================
diff --git a/.idea/.idea.Swashbuckler.dir/.idea/.gitignore b/.idea/.idea.Swashbuckler.dir/.idea/.gitignore
--- a/.idea/.idea.Swashbuckler.dir/.idea/.gitignore	(revision 3531baa5e048ad8429e6dfb12cf7ca335b408288)
+++ b/.idea/.idea.Swashbuckler.dir/.idea/.gitignore	(date 1761416273365)
@@ -11,3 +11,5 @@
 # Datasource local storage ignored files
 /dataSources/
 /dataSources.local.xml
+# obj folder
+/obj
\ No newline at end of file
Index: obj/rider.project.model.nuget.info
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>17614117345442710
===================================================================
diff --git a/obj/rider.project.model.nuget.info b/obj/rider.project.model.nuget.info
--- a/obj/rider.project.model.nuget.info	(revision 3531baa5e048ad8429e6dfb12cf7ca335b408288)
+++ b/obj/rider.project.model.nuget.info	(date 1761416114671)
@@ -1,1 +1,1 @@
-17614117345442710
\ No newline at end of file
+17614124441847868
\ No newline at end of file
